### DISCUSSION TITLE
有计划将Json序列化组件更换为System.Text.Json嘛

### DISCUSSION THREAD

**Discussion by heerheer** [Hoshikawa-Kaguya/Sora#40] [Q&A]
System.Text.Json似乎性能更好（

**Comment by Yukari316** [comment]
`System.Text.Json`的确性能更好，但是提升并不大
并且从`Newtonsoft.Json`迁移到`System.Text.Json`的话还是比较麻烦的，除了增加开发成本之外其实没什么好处
而且我自己最近也很忙应该是没时间写新的内容了( ´_ゝ｀)

**Reply by heerheer** [reply]
好耶，因为个人很喜欢System.Text.Json的Utf8JsonWriter( 微软真的是太懂了呜呜呜

**Reply by Yukari316** [reply]
可以来PR就有了（）
最近没太多时间做大的修改
